application_manager/plugins/chronos/plugin.py

```diff
@@ -40,7 +40,7 @@ def start_application(self, data):
             url = api.chronos_url
```

Comment:
I'm not sure the chronos plugin changes should be merged together with a pull request named Mesos-spark plugin implementation, but I won't block merging on this.